Makefile

```diff
@@ -245,7 +245,10 @@ $(SYSROOT_LIB)/libwasi-emulated-mman.a: $(LIBWASI_EMULATED_MMAN_OBJS)
 
 %.a:
 	@mkdir -p "$(@D)"
-	$(WASM_AR) crs $@ $^
+	$(WASM_AR) crs $@ $(wordlist 1, 200, $(patsubst $(CURDIR)/%.o,%.o,$^))
+	$(WASM_AR) crs $@ $(wordlist 200, 400, $(patsubst $(CURDIR)/%.o,%.o,$^))
+	$(WASM_AR) crs $@ $(wordlist 400, 600, $(patsubst $(CURDIR)/%.o,%.o,$^))
+	$(WASM_AR) crs $@ $(wordlist 600, 1000, $(patsubst $(CURDIR)/%.o,%.o,$^))
```

Comment:
Ooo.  This is pretty ugly.  Is this because of the command line limit?  I wonder if we can find some other way?  Perhaps a response file?    We should at least add a comment.  Also the final increment is bigger than the others, is that deliberate?

Reply:
`llvm-ar` doesn't take an input file, so I'm not sure there really is an alternative. I do certainly agree it's ugly :)

I added a comment explaining this, and also simplified and fixed things up a bit based on @sunfishcode's feedback.

Reply:
It looks like `llmv-ar` does take a response file but `--help` didn't document it.  I made an llvm change to add to the `--help` message: https://reviews.llvm.org/D63836.

I'm not totally sure it helps here but perhaps its an option.  Not a blocker for this change landing.